.eslintrc.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
       {
         singleQuote: true,
         arrowParens: 'avoid',
+        trailingComma: 'es5',
```

Comment:
We effectively have moved up to prettier v3.x with this PR, so I set trailingComma to the v2.x default value ("es5") to avoid lots of commas being added to lots of files :joy: